### PR TITLE
Blog - create write-up for Arcus.Scripting v0.1 announement

### PR DESCRIPTION
**Pull request by stijnmoreels**
Relates to https://github.com/arcus-azure/arcus.scripting/issues/74

### CODE REVIEW

File: community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Announcing Arcus Scripting
```
Comment (tomkerkhove) [inline community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:1]: Some general remarks:
- Looks good, but let's add some samples!

File: community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+This summary of all the features in this first version is only the beginning of a great scripting story.
+Here are some features that are on the horizon, but please don't heasitate to [make a suggestion](https://github.com/arcus-azure/arcus.scripting/issues/new?template=Feature_request.md) and we're happy to help where we can!
+
+* [Azure API Management](https://github.com/arcus-azure/arcus.scripting/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%3Aapi-management)
+  * [Provide script for restoring an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/76)
+  * [Provide script to backup an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/75)
+  * [Provide script to set the subscriptionKey on an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/39)
+* [Azure Cognitive Services](https://github.com/arcus-azure/arcus.scripting/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%3Acognitive-services)
+  * [Provide script set to build Azure Cognitive Search & Form Recognizer](https://github.com/arcus-azure/arcus.scripting/issues/22)
```
Comment (tomkerkhove) [inline community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:71]: I wouldn't include this one

File: community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md (hunk 1 above)
Comment (stijnmoreels) [reply community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:1]: Yes, done!

File: community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+This summary of all the features in this first version is only the beginning of a great scripting story.
+Here are some features that are on the horizon, but please don't heasitate to [make a suggestion](https://github.com/arcus-azure/arcus.scripting/issues/new?template=Feature_request.md) and we're happy to help where we can!
+
+* [Azure API Management](https://github.com/arcus-azure/arcus.scripting/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%3Aapi-management)
+  * [Provide script for restoring an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/76)
+  * [Provide script to backup an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/75)
+  * [Provide script to set the subscriptionKey on an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/39)
+* [Azure Key Vault](https://github.com/arcus-azure/arcus.scripting/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%3Akey-vault)
+  * [Provide script for importing certificate as base64-string into Azure Key Vault](https://github.com/arcus-azure/arcus.scripting/issues/71)
+
```
Comment (mbraekman) [inline community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:97]: I would include the planned scripts related to Azure Logic Apps.
This way, people can see that besides offering a test framework, we'll also be providing a set  of scripts to help them during deployment.
Something like:
 _Provide scripts to enable or disable logic apps in a fixed order, preventing data-loss during an upgrade._

File: community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+```powershell
+PS> Create-AzStorageTable -ResourceGroupName "stock" -StorageAccountName "admin" -TableName "products"
+# Creating table 'products' in the storage account 'admin'..."
+```
+
+See the [docs](https://scripting.arcus-azure.net/features/powershell/azure-storage) for more information on this function.
+
+## Azure Resource Manager (ARM)
+
+The `Arcus.Scripting.ARM` PowerShell module contains the capability to inject content into a ARM template, controling the formatting of the injected content.
```
Comment (mbraekman) [inline community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:69]: Perhaps include a bit of info as to what users will be able to use the module for in general?

```suggestion
The `Arcus.Scripting.ARM` PowerShell module has been built to help you in manipulating ARM templates before triggering a deployment, allowing you to reduce the complexity of these templates.  
Currently, this module contains the capability to inject content into an ARM template, controling the formatting of the injected content.
```

File: community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+```powershell
+PS> Set-AzDevOpsVariable -Name "Arcus.KeyVault.VaultUri" -Value "https://arcus.azure.vault.com"
+```
+
+See the [docs](https://scripting.arcus-azure.net/features/powershell/azure-devops) for more information on this function.
+
+## What's on the Horizon
+
+This summary of all the features in this first version is only the beginning of a great scripting story.
```
Comment (tomkerkhove) [inline community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:89]:
```suggestion
But this is only the beginning of a great scripting story!
```

File: community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+Here are some features that are on the horizon, but please don't heasitate to [make a suggestion](https://github.com/arcus-azure/arcus.scripting/issues/new?template=Feature_request.md) and we're happy to help where we can!
```
Comment (tomkerkhove) [inline community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:90]:
```suggestion
Here are some features that are on the horizon:
```

File: community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md (hunk 3 above)
Comment (tomkerkhove) [inline community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:100]:
```suggestion

Are we missing something? Don't hesitate to [make a suggestion](https://github.com/arcus-azure/arcus.scripting/issues/new?template=Feature_request.md) and we're happy to help where we can!
```

Reason is that it's best to show what we are planning and only then ask for more suggestions; this feels more natural to me.

Comment (stijnmoreels) [reply community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:100]: Aha, yes, added a list item for this. Thanks!

File: community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md (hunk 6 above)
Comment (stijnmoreels) [reply community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:90]: No suggestions for you 😄 .

Comment (stijnmoreels) [reply community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md]: Ah, nevermind, didn't see that.

**Comment by stijnmoreels** [comment]
> Some suggestions.
> 
> Would also be good to run it through https://www.grammarly.com/ as I see some spelling issues.

Grammerly doesn't work apparently in the edit window of files in GitHub. I know there are probably a lot of spelling errors. I guess that's also why this is only a draft and has to be re-read by someone.

**Comment by tomkerkhove** [comment]
> > Some suggestions.
> > Would also be good to run it through [grammarly.com](https://www.grammarly.com/) as I see some spelling issues.
> 
> Grammerly doesn't work apparently in the edit window of files in GitHub. I know there are probably a lot of spelling errors. I guess that's also why this is only a draft and has to be re-read by someone. I'm thinking about an editorial review when this is on WordPress.
> 
> Found some errors with another tool, though and fixed them.

If you go to https://app.grammarly.com/ and open a new doc you can validate it there.

File: community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+See the [docs](https://scripting.arcus-azure.net/features/powershell/azure-storage) for more information on this function.
+
+## Azure Resource Manager (ARM)
+
+The `Arcus.Scripting.ARM` PowerShell module has been built to help you in manipulating ARM templates before triggering a deployment, allowing you to reduce the complexity of these templates.  
+Currently, this module contains the capability to inject content into an ARM template, controlling the formatting of the injected content.
+After importing the module, the `Inject-ArmContent` function is available to you.
+
+The [docs](https://github.com/arcus-azure/arcus.scripting/blob/master/docs/preview/features/powershell/arm.md) has an example of an OpenAPI specifications you'd want to deploy.
```
Comment (mbraekman) [inline community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:73]:
```suggestion
The [docs](https://github.com/arcus-azure/arcus.scripting/blob/master/docs/preview/features/powershell/arm.md) show you an example of injecting OpenAPI specifications into an ARM template.
```

File: community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+
+But this is only the beginning of a great scripting story!
+Here are some features that are on the horizon:
+
+* [Azure API Management](https://github.com/arcus-azure/arcus.scripting/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%3Aapi-management)
+  * [Provide script for restoring an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/76)
+  * [Provide script to backup an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/75)
+  * [Provide script to set the subscriptionKey on an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/39)
+* [Azure Key Vault](https://github.com/arcus-azure/arcus.scripting/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%3Akey-vault)
+  * [Provide script for importing certificate as base64-string into Azure Key Vault](https://github.com/arcus-azure/arcus.scripting/issues/71)
```
Comment (mbraekman) [inline community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:97]:
```suggestion
  * [Provide script for importing a certificate as a base64-string into Azure Key Vault](https://github.com/arcus-azure/arcus.scripting/issues/71)
```

**Comment by stijnmoreels** [comment]
> > > Some suggestions.
> > > Would also be good to run it through [grammarly.com](https://www.grammarly.com/) as I see some spelling issues.
> > 
> > 
> > Grammerly doesn't work apparently in the edit window of files in GitHub. I know there are probably a lot of spelling errors. I guess that's also why this is only a draft and has to be re-read by someone. I'm thinking about an editorial review when this is on WordPress.
> > Found some errors with another tool, though and fixed them.
> 
> If you go to https://app.grammarly.com/ and open a new doc you can validate it there.

OK, done, added only a missing comma.

**Review by tomkerkhove — APPROVED** [review]

File: community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+  * [Provide script to backup an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/75)
+  * [Provide script to set the subscriptionKey on an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/39)
+* [Azure Key Vault](https://github.com/arcus-azure/arcus.scripting/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%3Akey-vault)
+  * [Provide script for importing a certificate as a base64-string into Azure Key Vault](https://github.com/arcus-azure/arcus.scripting/issues/71)
+* [Azure Logic Apps](https://github.com/arcus-azure/arcus.scripting/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%3Alogic-apps)
+  * Provide scripts to [enable](https://github.com/arcus-azure/arcus.scripting/issues/19) or [disable](https://github.com/arcus-azure/arcus.scripting/issues/20) logic apps in a fixed order, preventing data-loss during an upgrade
+
+Are we missing something? Don't hesitate to [make a suggestion](https://github.com/arcus-azure/arcus.scripting/issues/new?template=Feature_request.md) and we're happy to help where we can!
+Thanks for reading!
```
Comment (tomkerkhove) [inline community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:105]:
```suggestion

Thanks for reading!
```

File: community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+Arcus team
```
Comment (tomkerkhove) [inline community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:106]:
```suggestion

Arcus team
```

**Pull request merged**